openwisp_notifications/handlers.py

```diff
@@ -158,13 +157,23 @@ def send_email_notification(sender, instance, created, **kwargs):
     instance.save()
 
 
+@receiver(post_save, dispatch_uid='notification_related_object_cache')
+@receiver(post_delete, dispatch_uid='notification_related_object_cache_deleted')
```

Comment:
this is worrying to me, if we don't specify a sender, this invalidation will be triggered constantly, which would make all this work you're doing useless.

Maybe we can just rely on the timeout and decrease it to 2 days and avoid doing this at all.